tests/examples/test_examples.py

```diff
@@ -162,6 +162,8 @@ def test_mlflow_run_example(directory, params, tmp_path):
         ("transformers", [sys.executable, "whisper.py"]),
         ("sentence_transformers", [sys.executable, "simple.py"]),
         ("tracing", [sys.executable, "fluent.py"]),
+        ("tracing", [sys.executable, "client.py"]),
+        ("tracing", [sys.executable, "multithreading.py"]),
     ],
```

Comment:
note: the langchain example requires OpenAI API key so I didn't include it for auto-testing